packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -15,13 +15,98 @@ import { CONTACTS } from '../../contacts';
 export class VACrisisLineModal {
   @Element() el: HTMLElement;
 
+  /**
+   * Phone number for the crisis line. Defaults to 988.
+   */
+  @Prop() phoneNumber: string;
+
+  /**
+   * Text number for the crisis line. Defaults to 838255.
+   */
+  @Prop() textNumber: string;
+
+  /**
+   * URL for the chat service. Defaults to Veterans Crisis Line chat.
+   */
+  @Prop() chatUrl: string;
+
+  /**
+   * TTY number for the crisis line. Defaults to CONTACTS.CRISIS_TTY.
+   */
+  @Prop() ttyNumber: string;
+
+  /**
+   * Selector for an external button that should trigger the modal.
+   * When provided, the internal trigger button will not be rendered.
+   * Accepts ID (#myButton), class (.my-button), or attribute selector.
+   */
+  @Prop() triggerRef: string;
```

Comment:
This change is to handle the crisis line button, that is styled as a link, in the desktop footer on va.gov